scrapers/TransErotica.yml

```diff
@@ -0,0 +1,25 @@
+name: "TransErotica"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - transerotica.com/
+      - https://tour.transerotica.com/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
```

Comment:
details , date get also as extra results  the fields from the related scenes (more updates) in the page 
date gets also the duration when available
performers can be retrieved without relying on indexes

```yaml
  sceneScraper:
    common:
      $update: //div[@id="preview"]/following-sibling::div[@class="updateDetails"]
    scene:
      Title: //h1[@class="title_bar"]/text()
      Details:
        selector: $update/p[contains(.,"Description:")]
        postProcess:
          - replace:
            - regex: "^Description:"
              with:
      Date:
        selector: $update/p/span[@class="upddate"][contains(.,"/")]
        parseDate: 01/02/2006
      Performers:
        Name: $update/p/span[@class="tour_update_models"]/a
```